packages/@aws-cdk/aws-pipes-alpha/README.md

```diff
@@ -68,39 +66,18 @@ possible:
 - [Self managed Apache Kafka stream](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-kafka.html)
 - [Amazon SQS queue](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-sqs.html)
 
-> Currently no implementation exist for any of the supported sources. The following example shows how an implementation can look like. The actual implementation is not part of this package and will be in a separate one.
```

Comment:
Do all of these sources have implementations in the module today? If there are any that don't, it may be worth listing out the ones we support today, with plans to implement others later.

Reply:
I added a note below about the ones we currently support.